src/tex/appendix.tex

```diff
@@ -218,6 +218,8 @@ \subsection{Model performance} \label{sec:model_performance_app}
 
 \Cref{fig:performance_per_topic_tiny} shows the same analysis on \chembenchmini.
 
+\Cref{fig:performance_corpus_and_tiny} shows the performance of the models on the \chembench corpus and the \chembenchmini subset. The relative performance difference between both is actually quite similar across most models. This makes \chembenchmini subset a reliable subset for human baseline comparison and particularly valuable for rapid prototyping and initial model assessment phases.
```

Comment:
```suggestion
\Cref{fig:performance_corpus_and_tiny} shows the performance of the models on the \chembench corpus and the \chembenchmini subset. The relative performance difference between both is quite similar across most models. This makes \chembenchmini subset a reliable subset for human baseline comparison and particularly valuable for rapid prototyping and initial model assessment phases.
```

I think you also had some nice comments about the preference data - do you want to add them?

Reply:
added observation about chemical preference on GPT-4 and referenced breakdowns to tables in [da0c066](https://github.com/lamalab-org/chembench-paper/pull/69/commits/da0c0661be4d2e5dc30736e799e0c6503feb2b18)